src/movies.js

```diff
@@ -2,14 +2,6 @@ import fetch from 'node-fetch';
 
 const API_BASE = 'https://plankton-app-xhkom.ondigitalocean.app/api';
 
-export async function loadScreenings(id) {
-  const res = await fetch(
-    API_BASE + '/screenings?populate=movie&filters[movie]=' + id
-  );
-  const payload = await res.json();
-  return payload;
-}
```

Comment:
Good idea moving the `loadScreenings()` function to the `cmsAdapter`. Maybe it would have made sense to move other loading functions from here as well?